html/browsers/sandboxing/sandbox-disallow-popups.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>window.open in sandbox iframe</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/utils.js"></script>
+<body>
+<script>
+async_test(t => {
+  // check for message from the popup
+  const uuid = token();
+  const bc = new BroadcastChannel(uuid);
+  bc.onmessage = t.unreached_func('popup was opened');
+
+  // check for message from the iframe
+  window.onmessage = t.step_func(e => {
+    assert_equals(e.data, null, 'return value of window.open');
+    t.step_timeout(t.done, 1000); // allow some time to pass in case the popup was opened
```

Comment:
What is the goal of this?
You won't receive several posted message as far as I understand.

Reply:
The goal is to fail the test if the popup loaded despite `window.open` returning null.

Reply:
I don't see the popup using postMessage. As a result, I still think this step_timeout to be unnecessary.